src/pages/PicsPage.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import Pics from '../components/Pics';
+import loading from '../assets/images/loading-progress.gif';
+
+const PicsPage = () => {
+  const [tempPictList, setTempPictList] = useState([]);
+  const { pics } = useSelector((state) => state.pics);
+  const picList = Object.keys(pics);
+
+  const handleClick = (event) => {
+    if (event.target.value !== 'both') {
+      const pepito = picList.filter((date) => (pics[date].media === event.target.value));
+      setTempPictList(pepito);
+    } else {
+      setTempPictList(picList);
+    }
+  };
+
+  useEffect(() => {
+    setTempPictList(picList);
+  }, [pics]);
+
+  return (
+
+    <div className="totallist">
+      <form action="/" className="form" id="form1">
+        <p>Choose media type: image, video or both (default)</p>
+        <select id="select" name="select" onChange={handleClick}>
+          <option value="both">Both</option>
+          <option value="image">Image</option>
+          <option value="video">Video</option>
+        </select>
+      </form>
+
+      <ul className="piclist">
+        {
+        tempPictList.length ? tempPictList.map((date) => (
+          <Pics
```

Comment:
- [ ] Based on the requirements of this project, we are supposed to follow the design guidelines including:
- Typography: font face, size, and weight
- Layout: composition and space between elements. Kindly ameliorate your design to match the screenshot below.

Original Design | Your Implementation
:---:|:---:
![Screenshot from 2022-08-11 05-10-52](https://user-images.githubusercontent.com/63087491/184064062-8c11fde1-7df1-490d-82ec-d11615b0e183.png) | ![](https://user-images.githubusercontent.com/63087491/184064009-3a4643e8-c6bb-47f7-826d-5631c76a9e0e.png)

Reply:
Hi @FrederickMih 

Thank you for your review and advice. 

A comment on the screenshot you attached:

I rendered my app on my phone (and several friends and family phones) and desktop using the Netlify link and we all watch a perfect display. Please, see the attached screenshot. (We can not reproduce the un-ordered layout shown in the screenshot you attached. I'd like to know which media device or browser you used). 

![image](https://user-images.githubusercontent.com/100212928/184147734-602b6106-80b7-4924-ae28-67cc875f9651.png)

I really don't know what to do to improve the design, as I think that following the suggested pattern will, on the contrary, diminish the quality of the user experience, by distancing the app's look and feel from the spacial and NASA-related theme, colors, etc.

Thank you in advance for your consideration.

Reply:
Hi @DarioAlessioR, the way your app display locally was not the same as in Netify. Kindly fix the display locally to match that remotely.

Reply:
I think we can consider your design. Just fix the other raised issues

Thanks 🙏

Reply:
I added 2 new suites of tests, for a total of 15 tests.

Acknowledgment was credited to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio).